syntax="proto3";
import "graph.proto";

package pr_route;

message RouterSettings
{
	double bus_wait_time = 1;
	double bus_velocity = 2;
}

message Calculate
{
	bytes stopName = 1;
	bytes busNumber = 2;
	uint32 span_count = 3;
	double time = 4;
}